sdk/spring/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 4.2.0-beta.1 (Unreleased)
+## 4.2.0 (2022-05-25)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- This release is compatible with Spring Boot 2.5.0-2.5.13, 2.6.0-2.6.7. (Note: 2.5.x (x>13) and 2.6.y (y>7) should be supported, but they aren't tested with this release.)
+- This release is compatible with Spring Cloud 2020.0.3-2020.0.5, 2021.0.0-2021.0.2. (Note: 2020.0.x (x>5) and 2021.0.y (y>2) should be supported, but they aren't tested with this release.)
 
-### Other Changes
+### Dependency Updates
+Upgrade azure-sdk's version. Make them align to azure-sdk-bom:1.2.2 managed versions.
```

Comment:
Given not all of our underlying sdks are tracked in azure-sdk-bom, thus we have to mention those which are upgraded and not tracked by sdk bom, like `azure-spring-data-cosmos`

Reply:
Good point. Will update.